inputs/templates/test/Module09/Module09VisualizeSingleSample.json.tmpl

```diff
@@ -1,6 +1,6 @@
 {
         "Module09VisualizeSingleSample.sv_base_mini_docker" : "us.gcr.io/broad-dsde-methods/gatk-sv/sv-base-mini:mw-gnomad-02-6a66c96",
-        "Module09VisualizeSingleSample.sv_pipeline_rdtest_docker" : "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline-rdtest:mw-gnomad-02-6a66c96",
+        "Module09VisualizeSingleSample.sv_pipeline_docker" : "us.gcr.io/broad-dsde-methods/gatk-sv/sv-pipeline:2024-04-29-v0.28.4-beta-7fd43657",
```

Comment:
ping @mwalker174 Would you still prefer to use the `sv-pipeline-rdtest` image here or rather use the current latest build?

Reply:
Use the latest

Reply:
Sounds good, thanks!